pkg/agent/baker.go

```diff
@@ -258,11 +258,10 @@ func normalizeResourceGroupNameForLabel(resourceGroupName string) string {
 	if strings.HasSuffix(truncated, "-") ||
 		strings.HasSuffix(truncated, "_") ||
 		strings.HasSuffix(truncated, ".") {
-		if len(truncated) > 62 {
+		if len(truncated) > maxLen-1 {
```

Comment:
I can't seem to find where this is defined, can we name is something clearer like maxResourceGroupLabelLength or just maxKubernetesLabelLength